Misc/NEWS.d/next/Library/2023-04-28-19-08-50.gh-issue-103977.msF70A.rst

```diff
@@ -0,0 +1 @@
+Improve import time of platform module.
```

Comment:
```suggestion
Improve import time of :mod:`platform` module.
```

Reply:
Should I ship a bot for this?

<img width="562" alt="Screenshot 2023-04-29 at 3 21 09 PM" src="https://user-images.githubusercontent.com/12621235/235326436-1c0e0123-a038-4f7f-8862-03a6ec6324f4.png">